package.json

```diff
@@ -24,17 +24,17 @@
     "get-started": "npm i && npm run start --prefix tools/getting-started",
     "help": "nx help",
     "lint": "npm run affected:lint",
-    "lint:file-folder-convention": "node ./tools/enforce-file-folder-naming-convention.js",
+    "lint:file-folder-convention": "node ./tools/enforce-file-folder-naming-convention.ts",
     "mongo": "run-script-os",
     "mongo:default": "sudo docker-compose -f ./apps/database/mongo/docker-compose.yml up",
     "mongo:win32": "docker-compose -f ./apps/database/mongo/docker-compose.yml up",
     "nx": "nx",
     "postgres": "run-script-os",
     "postgres:default": "sudo docker-compose -f ./apps/database/postgres/docker-compose.yml up",
     "postgres:win32": "docker-compose -f ./apps/database/postgres/docker-compose.yml up",
-    "remap-api-url": "node ./.do/remap-api-url.js",
-    "remap-redoc": "node ./.do/remap-redoc.js",
-    "rename-project": "node ./tools/rename-project.js",
+    "remap-api-url": "node ./.do/remap-api-url.ts",
```

Comment:
node can directly run ts files?
I would highly doubt that: https://stackoverflow.com/questions/33535879/how-to-run-typescript-files-from-command-line